cmd/neofs-node/object.go

```diff
@@ -327,7 +327,7 @@ func initObjectService(c *cfg) {
 	)
 
 	// build service pipeline
-	// grpc | <metrics> | acl | signature | response | split
```

Comment:
Could you explain why this change?
It seems like the previous order was correct.

Reply:
maybe i'm wrong, did not look deeply, just saw the order of wrapping and saw this commits:

1. that comment [appeared](https://github.com/nspcc-dev/neofs-node/commit/534ce034217d3d4bd623484309340998e1a76408);
2. then the order was [changed](https://github.com/nspcc-dev/neofs-node/commit/e74ff7c15a770570ae8318399de1b8c35fc8f384) without fixing the comment

Reply:
I don't think this comment is really useful.